src/backend/main.ts

```diff
@@ -1150,6 +1159,10 @@ ipcMain.handle(
       status: 'playing'
     })
 
+    if (minimizeOnGameLaunch) {
+      mainWindow?.hide()
+    }
+
```

Comment:
Actually none of this makes any difference in the end since the launch command was not even called.
I realized that we minimize the window just after the first call from protocol and was trying to avoid that, but the minimize command always happens when calling from there and I think we should not change this behavior.

Reply:
got the line back to where it was.